app/src/main/java/com/keylesspalace/tusky/components/timeline/TimelinePagingAdapter.kt

```diff
@@ -59,7 +59,7 @@ class TimelinePagingAdapter(
             }
             else -> {
                 val view = LayoutInflater.from(viewGroup.context)
-                    .inflate(R.layout.item_status, viewGroup, false)
+                    .inflate(R.layout.item_status_wrapper, viewGroup, false)
```

Comment:
Wouldn't it be better to have another viewtype "VIEW_TYPE_FILTERED_STATUS" instead? (less stuff to inflate, less layouts hanging around at runtime)

Reply:
🤔 If we do that, will we still be able to dynamically show the status when the user clicks "show anyway" ?

Reply:
I think so

Reply:
Oh, *now* I see what you mean - I completely misinterpreted what you meant before 🤦